src/main/java/ch/njol/skript/expressions/base/PropertyExpression.java

```diff
@@ -70,8 +71,8 @@ protected final void setExpr(final Expression<? extends F> expr) {
 	}
 	
 	@Override
-	protected final T[] get(final Event e) {
-		return get(e, expr.getArray(e));
+	protected final T[] get(final Event event) {

```

Comment:
I'm still waiting for explanation why e -> event in places that have not been otherwise changed.

Reply:
I was fixing a null pointer there, but it should be handled by a class that extends PropertyExpression